src/dune_engine/scheduler.ml

```diff
@@ -63,7 +63,7 @@ let interrupt_signals : Signal.t list = [ Int; Quit; Term ]
 
 (* In addition, the scheduler also blocks some other signals so that only
    designated threads can handle them by unblocking *)
-let blocked_signals : Signal.t list = Winch :: interrupt_signals
+let blocked_signals : Signal.t list = Cont :: Tstp :: Winch :: interrupt_signals
```

Comment:
Can we document some reasons for why certain signals are in this list, maybe linking to the corresponding issues?

Reply:
There's no particular issues (because nobody is using tui yet), but I did leave a comment for each and every single describing the need to block it.